src/main/java/org/ladocuploader/app/csv/model/StudentCsvModel.java

```diff
@@ -102,13 +106,18 @@ public static List<BaseCsvModel> generateModel(Submission submission){
 
         if (household != null && !household.isEmpty()) {
             for (Map<String, Object> member : household) {
-                String relationship = (String) member.get("householdMemberRelationship");
-                if (relationship == null || !relationship.equalsIgnoreCase("child")) {
+                //String relationship = (String) member.get("householdMemberRelationship");
```

Comment:
I think you can take this commented out code out -- it was replaced by the "isMemberEceEligible" code.

Reply:
If we are mapping this object, the applicant was interested in ECE, so the "isMemberEceEligible" will check to see if the household member is either pregnant or young enough to be the student.  If so, it will add them as a student.